qubesmanager/settings.py

```diff
@@ -220,6 +220,14 @@ def __init__(self, vm, init_page="basic", qapp=None, qubesapp=None,
         ####### apps tab
         if self.tabWidget.isTabEnabled(self.tabs_indices["applications"]):
             self.app_list = multiselectwidget.MultiSelectWidget(self)
+            if self.vm.template:
+                self.app_list.change_labels(
+                    available="Applications available in template",
```

Comment:
The list for template-based qubes includes apps from both its template and the app qube itself (if you have something installed in your home dir). So, a common text for both cases sounds better.